### PR TITLE
feat: Add kwargs for explicit data passing in gaze.from_numpy()

### PR DESCRIPTION
**Pull request by dkrako**
## Description

Adds new keyword arguments to pass time, position, velocity numpy arrays as keyword arguments to `pm.gaze.from_numpy()`

Fixes issue #501

## Implemented changes

- [x] Make `data` and `schema` argument optional
- [x] Add `time`, `pixel`, `position`, `velocity`, `acceleration` arguments

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change is or requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
Provide instructions so we can reproduce.
Please also list any relevant details for your test configuration

- [x] Test full specification with schema
- [x] Test full specification with explicit lists
- [x] Test all `None`

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/pymovements/gaze/integration.py [hunk 1]
```diff
@@ -69,7 +85,59 @@ def from_numpy(
     -------
     py:class:`~pymovements.GazeDataFrame`
     """
-    df = pl.from_numpy(data=data, schema=schema, orient=orient)
+    if data is not None:
+        checks.check_is_mutual_exclusive(data=data, time=time)
```
Comment (prassepaul) [inline src/pymovements/gaze/integration.py:89]: add a comment what is tested in the lines...

**Review by prassepaul — CHANGES_REQUESTED** [review]
I would add some more comments (can be high-level)...

File: src/pymovements/gaze/integration.py (hunk 1 above)
Comment (dkrako) [reply src/pymovements/gaze/integration.py:89]: Thanks, good idea!

I will also add an example for each case to the docstring.

Comment (dkrako) [reply src/pymovements/gaze/integration.py]: done

**Review by prassepaul — APPROVED** [review]
thx

**Pull request merged**